src/plays/language-translater/styles.css

```diff
@@ -0,0 +1,135 @@
+/* enter stlyes here */
+:root {
+  --white: white;
+}
+
```

Comment:
please avoid targeting directly to the `root` it can break the whole website, instead add a separate name for the class.